.ci/setup.sh

```diff
@@ -70,6 +70,16 @@ enable_nested_virtualization() {
 	aarch64 | ppc64le)
 		info "CI running in bare machine"
 		;;
+	s390x)
+		if [ $(systemd-detect-virt) == "kvm" ]; then
+			info "Running in kvm guest. Nested KVM is needed"
+			sudo modprobe -r kvm
+			sudo modprobe kvm nested=1
+			if [ grep -q "N" /sys/module/kvm/parameters/nested 2>/dev/null ]; then
+				die "Failed to find or enable Nested virtualization"
```

Comment:
This block looks almost identical to `enable_nested_virtualization()` in `.ci/setup.sh`. Could you just update that to handle os390x?

Reply:
sure, i can do that